mappings/net/minecraft/entity/passive/PufferfishEntity.mapping

```diff
@@ -2,10 +2,13 @@ CLASS net/minecraft/class_1454 net/minecraft/entity/passive/PufferfishEntity
 	FIELD field_30353 NOT_PUFFED I
 	FIELD field_30354 SEMI_PUFFED I
 	FIELD field_30355 FULLY_PUFFED I
+	FIELD field_33692 BLOW_UP_PREDICATE Lnet/minecraft/class_4051;
```

Comment:
How does this differ from `BLOW_UP_FILTER`?

Relates to #1624

Reply:
guess this is a target predicate?

Reply:
```suggestion
	FIELD field_33692 BLOW_UP_TARGET_PREDICATE Lnet/minecraft/class_4051;
```